custom_model_runner/datarobot_drum/drum/adapters/model_adapters/python_model_adapter.py

```diff
@@ -584,15 +584,16 @@ def _predict_new_drum(self, data, **kwargs) -> RawPredictResponse:
     def _predict_legacy_drum(self, data, model, **kwargs) -> RawPredictResponse:
         positive_class_label = kwargs.get(POSITIVE_CLASS_LABEL_ARG_KEYWORD)
         negative_class_label = kwargs.get(NEGATIVE_CLASS_LABEL_ARG_KEYWORD)
-        request_labels = (
-            get_request_labels(
+        if self._target_type in {TargetType.BINARY, TargetType.MULTICLASS}:
+            request_labels = get_request_labels(
                 kwargs.get(CLASS_LABELS_ARG_KEYWORD),
                 positive_class_label,
                 negative_class_label,
             )
-            if self._target_type in {TargetType.BINARY, TargetType.MULTICLASS}
-            else None
-        )
+        elif self._target_type == TargetType.GEO_POINT:
+            request_labels = ["latitude", "longitude"]
```

Comment:
Consider making new enums for geo point request labels, which would be added in:
```
custom_model_runner/datarobot_drum/drum/enum.py
```